src/tests/multihost/alltests/test_kcm.py

```diff
@@ -52,3 +52,37 @@ def test_client_timeout(self, multihost, backupsssdconf):
                                                    " /var/log/sssd/"
                                                    "sssd_kcm.log")
         assert 'Terminated client' in grep_cmd.stdout_text
+
+    @pytest.mark.no_tier
+    def test_kcm_check_socket_path(self, multihost, backupsssdconf):
+        """
+        :Title: kcm: Test socket path when sssd-kcm is activated by systemd
+        :id: 6425bf2c-d07e-4d65-b15d-946141422f96
+        ticket: #https://github.com/SSSD/sssd/issues/5406
+        """
+        # Start from a known-good state after removing log file and adding a
+        # new socket path
+        domain_log = '/var/log/sssd/sssd_kcm.log'
+        multihost.client[0].service_sssd('stop')
+        multihost.client[0].run_command("systemctl stop sssd-kcm")
+        server = sssdTools(multihost.client[0])
```

Comment:
Creating an instance with server for a client, will be confusing. Please use client or client_libs for use on a client.